test/src/main/java/org/apache/accumulo/test/WaitForBalanceIT.java

```diff
@@ -115,13 +126,13 @@ private boolean isBalanced(AccumuloClient c) throws Exception {
       return false;
     }
     int average = 0;
-    for (Integer i : counts.values()) {
+    for (Integer i : tserverCounts.values()) {
       average += i;
     }
-    average /= counts.size();
-    System.out.println(counts);
+    average /= tserverCounts.size();
```

Comment:
Computing the average seems like it should use a double.  This is an existing problem and not sure if even matters.